build.sh

```diff
@@ -16,6 +16,11 @@ TAG="nightly"
 TARGET="registry"
 USE_DIGESTS=false
 DOCKERFILE="./build/dockerfiles/Dockerfile"
+if [ "$(uname -m)" == "s390x" ]; then
+    REGISTRY_IMAGE="registry.redhat.io/rhel8/httpd-24:latest"
+else
+    REGISTRY_IMAGE="registry.centos.org/centos/httpd-24-centos7"
```

Comment:
riddle me this... why can't we use rhel8 instead of centos for both arches? @benoitf since neither is alpine or ubuntu can we just switch to RHEL8 instead of Centos7? Why be based on two completely different evolutions of the same OS?

Reply:
it will use alpine/httpd like in plugin registry so this changé will be removed